stix_shifter/stix_transmission/src/modules/security_advisor/security_advisor_connector.py

```diff
@@ -0,0 +1,30 @@
+from ..base.base_connector import BaseConnector
+from .security_advisor_ping import SecurityAdvisorPing
+from .security_advisor_query_connector import SecurityAdvisorQueryConnector
+from .security_advisor_status_connector import SecurityAdvisorStatusConnector
+from .security_advisor_results_connector import SecurityAdvisorResultsConnector
+from .security_advisor_delete_connector import SecurityAdvisorDeleteConnector
+from .security_advisor_auth import SecurityAdvisorAuth
+
+class Connector(BaseConnector):
+    def __init__(self, connection, configuration ):
+        
+        auth  = {}
+        accountID = configuration.get("ibmCloudAccountID")
+        apiKey = configuration.get("ibmCloudApiKey") 
+        host = configuration.get("saAPIEndpoint")
```

Comment:
This should be connection.get("host"). This is how the endpoint will be stored in the connection object.

Reply:
We dont have a separate endpoint .  We just need above configurations to make connection. Please see datasource PR.